repository/src/test/resources/alfresco/transforms/0999-testPipeline.json

```diff
@@ -1,16 +1,17 @@
 {
```

Comment:
Modifications to get tests to pass, now that we throw out pipelines with invalid steps.
Used "overrideSupported" and "addSupported" (which is nice as the repo was not using them)on the step transformers in the original "iWorksToImageViaJpg" pipeline which the "rubbish" one (now removed) was based.